whois-commons/src/test/java/net/ripe/db/whois/common/rpsl/transform/FilterAuthFunctionTest.java

```diff
@@ -189,50 +187,13 @@ public void crowd_client_exception() {
         assertThrows(AuthServiceClientException.class, () -> {
             final UserSession userSession = new UserSession("d06e5500-ac91-4336-94f3-76cab38b73eb","user@host.org", "Test User", true, "2033-01-30T16:38:27.369+11:00");
 
-            when(ssoTokenTranslator.translateSsoToken("token")).thenReturn(userSession);
             when(authServiceClient.getUsername("d06e5500-ac91-4336-94f3-76cab38b73eb")).thenThrow(AuthServiceClientException.class);
 
-            subject = new FilterAuthFunction(Collections.<String>emptyList(), null, "token", ssoTokenTranslator, authServiceClient, rpslObjectDao, Lists.newArrayList(), clientAuthCertificateValidator);
+            subject = new FilterAuthFunction(Collections.<String>emptyList(), null, userSession, authServiceClient, rpslObjectDao, Lists.newArrayList(), clientAuthCertificateValidator);
             subject.apply(RpslObject.parse("" +
                     "mntner: SSO-MNT\n" +
                     "auth: SSO d06e5500-ac91-4336-94f3-76cab38b73eb\n" +
                     "source: RIPE"));
         });
     }
-
-    @Test
```

Comment:
Can you keep these tests?

Reply:
These test cases were throwing authserviceException which cannot happen anymore as userssion is now passed directly